src/drivers/angular2.js

```diff
@@ -0,0 +1,99 @@
+import { omit, replaceObject } from '../lib';
+/* eslint-disable new-cap, object-shorthand */
+
+
+export let angular2 = {
+
+    global() {
+        return (window.ng && window.ng.core && window.ng.platformBrowser) ? 
+        {
+            Component: window.ng.core.Component,
+            NgModule: window.ng.core.NgModule,
+            ElementRef: window.ng.core.ElementRef,
+            NgZone: window.ng.core.NgZone,
+            BrowserModule: window.ng.platformBrowser.BrowserModule
+        } 
+        : false;
+    },
+
+    register(xcomponent, { Component, NgModule, ElementRef, NgZone, BrowserModule }) {
+
+        xcomponent.log('initializing angular2 component');
+        
+        const getBindingMetadata = () => {
+            const inputs = [];
+            if (xcomponent.looseProps) {
+                inputs.push('props');
+            } else {
+                for (let key of Object.keys(xcomponent.props)) {
+                    inputs.push(key);
+                }
+            }
+            return { inputs };
+        };
+
+
+        function getProps(component) {
+            const props = component.props || omit(component, ['$xContext']);
```

Comment:
We dont have a list of known props in loose props mode. Unless if we only communicate via a single props object or dumb loose props support list of known props is not applicable here.